src/lib/types.ts

```diff
@@ -0,0 +1,21 @@
+export interface Car { // Car object fields
+    maker: string
+    model: string
+    year: string
+    engine: string
+}
+
+export interface Motorcycle {
+    maker: string;
+    model: string;
+    year: number;
+    engine: string;
+}
+
+export const makers = ['Honda', 'Yamaha', 'Suzuki', 'Kawasaki', 'Ducati'];
+
+export const engineTypes = ['Petrol', 'Diesel', 'Electric'] // Engine types
+
+export interface FormErrors extends Car {}
+
+export const emptyCar: Car = { maker: '', model: '', year: '', engine: '' } // Empty car object
```

Comment:
Since your file name is `types`, it should contain only interfaces or types. Please, move all the constants to the components where they are used.

It's nice approach to define constants in a separate files, but for cases such as `makers`, that is array which only used in one component, it's okay to define it in the component itself.